src/main/java/edu/rice/habanero/benchmarks/BenchmarkRunner.java

```diff
@@ -25,15 +27,9 @@ private static void parseArgs(final String[] args) throws Exception {
             System.setProperty("actors.maxPoolSize", numWorkers);
         }
 
-        final int argLimit = args.length - 1;
-        for (int i = 0; i < argLimit; i++) {
-            final String argName = args[i];
-            final String argValue = args[i + 1];
-
-            if (argName.equalsIgnoreCase("-iter")) {
-                iterations = Integer.parseInt(argValue);
-            }
-        }
+        CliArgumentParser argParser = new CliArgumentParser(args);
+        iterations = argParser.getValue(new String[]{"-iter", "--iterations"}, iterations);
+        no_evaluation = argParser.getValue(new String[]{"--no_evaluation"}, no_evaluation);
```

Comment:
Should we instead call this `simple` and have a corresponding variable `simpleExecutionRequested`?